src/util/RNG.cpp

```diff
@@ -23,7 +23,7 @@ std::vector<std::unique_ptr<util::RNG>> splitEngine(
   std::vector<util::RNG::result_type> improvedSeeds(_numOutputs);
   seqSeeds.generate(std::begin(improvedSeeds), std::end(improvedSeeds));
 
-  // Create the 
+  // Create the
```

Comment:
It would be good if someone added this line finish it. :smile: Or, I'll try to do my best to guess what was it.